Dockerfile.prod

```diff
@@ -0,0 +1,21 @@
+FROM node:latest as builder
+
+WORKDIR /server
+
+RUN git clone https://github.com/codeuino/social-platform-donut-frontend.git
+
+WORKDIR /server/social-platform-donut-frontend
+
+RUN npm install && \
```

Comment:
be verbose with npm version

Reply:
Are you pointing to the tag `--loglevel` ?

Reply:
https://medium.com/@Quigley_Ja/specifying-npm-version-in-dockerfile-ee6e71a60945

Reply:
@vaibhavdaren Thanks for the link, I have changed node from LTS to 14 so this will be always using a *specific version* of npm which is v6 (Major). 
However, the link mentions about using a higher version of npm outside the lower node versions. I guess now we have got v14 (three years later) with stringent npm v6 within the container. So. we no need to install again npm as per the link. Does this sound right?